### ISSUE TITLE
How to generate context_params for the test set on the srn cars?

### ISSUE THREAD

**Issue by 1999kevin** [tackgeun/mNIF#1]
Hi, I find that the auto-decoding procedure computes context vectors for the training set during stage 1 training for srn cars. Then how to generate context_params for the test set?

**Comment by tackgeun** [comment]
In auto-decoding procedure, you need to optimize the new context vectors using test set. Starting from the pre-trained mNIF from SRN Cars training set, optimize only context vector with randomly initialized one. 
In code, you need to set --lr_outer=0.0 and --lr_inner=1e-4 (for instance)
